packages/common-all/src/constants/configs/commands.ts

```diff
@@ -53,6 +53,10 @@ const NOTE: DendronConfigEntryCollection<NoteLookupConfig> = {
     label: "Confirm Vault on Create.",
     desc: "Prompt for vault selection when creating new note.",
   },
+  selectVaultModeOnCreate: {
+    label: "Vault selection mode on note creation.",
+    desc: "When new note is created this configures whether Dendron should smartly guess the vault for the note or always prompt to select a vault. Values:[smart, alwaysPrompt].",
+  },
```

Comment:
super nit: these descriptions are not used anywhere yet, but will be used in a webview for config in the future. can we make the tone consistent with the others?

1. Consistent capitalization of label
1. short form description with consistent tone with others.
1. remove description of possible values in the `desc` field as these will be provided by the to-be-implemented webview as a dropdown form.

```suggestion
  selectVaultModeOnCreate: {
    label: "Vault Selection Mode on Create.",
    desc: "Sets what vault selection mode should be used when creating new note.",
  },
```

Reply:
while we are at it, can we also fix the label and description for `fuzzThreshold`?

```
fuzzThreshold: {
  label: "Fuzziness Threshold for Lookup",
  desc: "Sets the fuzziness level of lookup".
}
```